test/csuite/random_directio/util.c

```diff
@@ -105,7 +106,7 @@ copy_directory_int(const char *fromdir, const char *todir, bool directio)
          */
         if (buf == NULL) {
             if (directio) {
-                blksize = (size_t)WT_MAX(sb.st_blksize, WT_BUFFER_ALIGNMENT_DEFAULT);
+                blksize = (size_t)WT_MAX(sb.st_blksize, BUFFER_ALIGNMENT_DEFAULT);
```

Comment:
Should the directio parameter and this if be deleted?

Reply:
Hi Tod, good question - no it doesn't need removal because the directio used in random_directio is not the same as the api config direct_io we are removing.